torch_geometric/nn/pool/topk_pool.py

```diff
@@ -57,18 +58,24 @@ def topk(
         mask = torch.cat(mask, dim=0)
 
         perm = perm[mask]
+    else:
+        raise ValueError(
+            "at least one of the min_score and ratio parameters must be "
+            "specified")
 
     return perm
 
 
-def filter_adj(edge_index, edge_attr, perm, num_nodes=None):
+def filter_adj(
+        edge_index: Tensor, edge_attr: Optional[Tensor], perm: Tensor,
+        num_nodes: Optional[int] = None) -> Tuple[Tensor, Optional[Tensor]]:
     num_nodes = maybe_num_nodes(edge_index, num_nodes)
 
     mask = perm.new_full((num_nodes, ), -1)
     i = torch.arange(perm.size(0), dtype=torch.long, device=perm.device)
     mask[perm] = i
 
-    row, col = edge_index
+    row, col = edge_index[0], edge_index[1]
```

Comment:
Hi, I changed this line because it raised following error while jitting:
```
./test/nn/pool/test_topk_pool.py::test_filter_adj Failed: [undefined]RuntimeError: 
Tensor cannot be used as a tuple:
  File "/home/camille/dev/pytorch_geometric/torch_geometric/nn/pool/topk_pool.py", line 78
    mask[perm] = i

    row, col = edge_index
               ~~~~~~~~~~ <--- HERE
    row, col = mask[row], mask[col]
    mask = (row >= 0) & (col >= 0)
```